### DISCUSSION TITLE
Titanium Production Build - After March 2022

### DISCUSSION THREAD

**Discussion by andrespompiglio** [appcelerator/titanium_mobile#13307] [General]
Hi,
We have to apply a fix to an old legacy version of a titanium App. as of march 1st our axway account was discontinuated (because of appcelerator discontinuating the product...)

This is our current configuration:

Mac OS Version: El Capitan
Appcelerator studio 4.8.1
Titanium SDK 5.5.0
Titanium CLI 4.1.5
Node 8
Appcelerator Package 7.0.3

 
Problems:

1) When we trying to login to Appcelerator Studio (or appc login) it says: Bad response from login (incorrect user data), please try again.

     a. Is there a way to tell Appcelerator Studio to skip login? so we can debug the application

 2) When trying to build a production package (.ipa) with appc run it says: Your current login session is not entitled to package mobile app. Please login to a different organization with entitlements or upgrade to a plan that supports production builds.

   a. We've tried to build the app by using titanium build command but it says: 

Running .. Titanium/mobilesdk/osx/5.5.0.GA/iphone/titanium_prep ...

Error: write EPIPE
  at _errnoException (util.js:992:11)
  at WriteWrap.afterWrite (net.js:864:14)

**Comment by cb1kenobi** [comment]
@andrespompiglio Hello! Sorry, after March 1, 2022, Appcelerator Studio and the Appcelerator CLI will no longer work.

You must switch to the Titanium CLI and an editor of your choosing as described here: [https://github.com/m1ga/from_zero_to_app/blob/master/appc_to_ti.md](https://github.com/m1ga/from_zero_to_app/blob/master/appc_to_ti.md)

**Reply by m1ga** [reply]
and also update your system: node 14, `sudo npm i -g titanium alloy` and then move to ti sdk 9.3.2 or better 10.1.1.

**Comment by jasonkneen** [comment]
Really? 5.5.0 sdk? That's 6 years old. 

You should really have been updating to SDKs as they were released and updating CLI tools too. 

There's been so much notice over the last year about the appc CLI ending and it's been recommended for years to drop studio for VSCode and the Ti extension. 

You may need some paid help to resolve this as apply one simple fix could now mean a LOT of work updating the app for latest SDKs; Xcode and Android versions etc. 

It's unlikely you'll be able to build anything with your setup and more than likely you can't submit to AppStore with the Xcode you're probably having to use with a 6 year old OS.

**Reply by jasonkneen** [reply]
Suggest posting in the jobs channel in TiSlack for help.

**Comment by maicojc** [comment]
because of that i'm going to another platform... for me, Titanium was died with this,  now welcome Flutter

**Reply by caspahouzer** [reply]
Because of what? If somebody thinks he doesn't have to do any work on an app made 6 years before? Do you think flutter would have a seamless update after such a period?
